deepchecks/vision/__init__.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from .dataset import VisionDataset
+from .base import (
+    Context,
+    Check,
+    Suite,
+    SingleDatasetBaseCheck,
+    TrainTestBaseCheck,
+    ModelOnlyBaseCheck
+)
+
+logger = logging.getLogger("deepchecks")
+
+try:
+    import torch
+except ImportError:
+    logger.error("PyTorch is not installed. Please install it in order to use deepchecks.vision")
+
```

Comment:
we should reraise the `ImportError`, not just log the message, 
also would be better to change the message to something like
`deepchecks was not installed with a set of dependencies to support computer vision functionality, please run pip install deepchecks[vision]`

Reply:
@ItayGabbay what's your take? The message @yromanyshyn isn't 100% accurate as installing deepchecks[vision] won't install pytorch (user must do that on his own), but it's a legit option to just raise an error (as nothing in the package would work if he tries it later on).

Reply:
I'll change it to ImportError